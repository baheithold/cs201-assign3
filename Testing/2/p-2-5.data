11 19 339 ; 0 2 179 ; 17 20 186 ; 7 30 55 ; 12 23 229 ; 
1 4 8 ; 1 3 350 ; 16 17 192 ; 24 7 140 ; 7 21 449 ; 
26 16 214 ; 29 23 477 ; 17 14 493 ; 15 13 251 ; 6 10 438 ; 
10 30 178 ; 13 5 428 ; 30 13 345 ; 5 1 149 ; 26 2 125 ; 
11 20 192 ; 2 17 470 ; 16 15 187 ; 29 30 412 ; 7 20 71 ; 
6 23 1 ; 11 27 115 ; 24 0 497 ; 25 21 447 ; 27 2 373 ; 
16 22 359 ; 1 4 42 ; 16 16 453 ; 14 12 129 ; 3 24 479 ; 
26 9 230 ; 21 5 392 ; 21 14 167 ; 10 21 247 ; 12 2 476 ; 
2 10 452 ; 6 28 240 ; 22 17 317 ; 3 23 30 ; 27 26 276 ; 
4 4 199 ; 9 21 125 ; 23 16 186 ; 13 4 227 ; 15 16 45 ; 
25 24 469 ; 22 16 339 ; 7 10 126 ; 30 20 368 ; 24 12 331 ; 
29 8 438 ; 18 29 339 ; 2 8 352 ; 6 7 328 ; 23 17 361 ; 
15 6 28 ; 0 30 190 ; 10 28 388 ; 30 16 75 ; 10 8 86 ; 
18 26 398 ; 16 17 95 ; 24 18 217 ; 31 10 432 ; 16 15 338 ; 
23 3 265 ; 21 17 165 ; 17 21 129 ; 1 1 327 ; 9 11 141 ; 
4 13 347 ; 21 13 146 ; 7 30 366 ; 17 18 490 ; 0 13 301 ; 
4 30 494 ; 21 31 185 ; 11 0 270 ; 12 11 23 ; 23 20 164 ; 
4 8 26 ; 17 14 398 ; 16 4 471 ; 2 18 247 ; 15 31 256 ; 
14 29 361 ; 13 18 293 ; 14 28 390 ; 25 23 41 ; 14 20 25 ; 
22 26 76 ; 4 4 445 ; 8 3 430 ; 26 10 46 ; 25 3 429 ; 
23 28 416 ; 9 25 274 ; 6 11 168 ; 29 9 226 ; 17 18 275 ; 
11 19 161 ; 15 24 379 ; 19 14 427 ; 29 21 23 ; 0 30 157 ; 
28 22 483 ; 21 29 494 ; 1 17 227 ; 10 19 453 ; 28 14 47 ; 
15 24 320 ; 7 23 4 ; 22 27 495 ; 11 6 414 ; 9 12 64 ; 
31 11 200 ; 28 1 185 ; 13 0 162 ; 0 6 23 ; 15 9 161 ; 
7 29 102 ; 30 29 273 ; 25 13 269 ; 0 15 371 ; 12 15 330 ; 
23 16 75 ; 24 13 374 ; 24 31 3 ; 30 19 480 ; 7 16 181 ; 
5 10 84 ; 2 20 499 ; 15 5 157 ; 31 12 487 ; 14 30 363 ; 
30 25 443 ; 11 19 62 ; 11 10 418 ; 30 15 386 ; 14 20 125 ; 
25 8 32 ; 13 3 77 ; 18 28 320 ; 30 15 51 ; 28 2 390 ; 
21 1 401 ; 8 23 135 ; 19 5 279 ; 2 12 346 ; 22 17 360 ; 
30 29 334 ; 1 15 75 ; 29 3 4 ; 13 6 242 ; 15 9 160 ; 
17 26 440 ; 8 9 361 ; 14 4 94 ; 26 10 493 ; 11 14 243 ; 
8 1 173 ; 23 12 219 ; 26 5 232 ; 1 15 369 ; 10 24 470 ; 
4 28 398 ; 13 22 447 ; 18 24 179 ; 28 13 217 ; 11 4 95 ; 
12 17 142 ; 24 28 39 ; 29 8 66 ; 13 17 45 ; 5 8 26 ; 
1 14 104 ; 23 5 328 ; 15 4 37 ; 28 18 363 ; 0 25 152 ; 
17 12 263 ; 13 7 235 ; 21 25 143 ; 7 6 493 ; 15 20 26 ; 
29 12 255 ; 2 8 120 ; 6 18 188 ; 24 13 199 ; 18 5 0 ; 
30 8 332 ; 5 23 222 ; 30 31 387 ; 5 4 61 ; 25 22 112 ; 
6 13 11 ; 14 5 360 ; 0 21 402 ; 14 21 20 ; 19 17 285 ; 
28 13 439 ; 19 27 329 ; 18 23 142 ; 22 8 296 ; 12 0 189 ; 
25 7 347 ; 31 11 373 ; 20 17 299 ; 9 29 164 ; 26 2 330 ; 
8 25 449 ; 3 14 341 ; 26 8 37 ; 2 1 383 ; 2 30 22 ; 
9 21 182 ; 20 18 268 ; 5 30 251 ; 3 10 171 ; 5 4 377 ; 
31 17 71 ; 13 27 87 ; 22 24 295 ; 23 7 285 ; 21 13 120 ; 
11 0 58 ; 29 12 316 ; 27 23 252 ; 5 22 270 ; 9 31 241 ; 
27 3 450 ; 22 14 41 ; 14 29 491 ; 21 7 293 ; 3 19 448 ; 
3 1 293 ; 15 25 176 ; 6 6 174 ; 28 24 85 ; 28 13 75 ; 
31 7 472 ; 13 1 431 ; 11 8 293 ; 18 5 243 ; 5 25 17 ; 
6 2 488 ; 0 23 339 ; 6 9 42 ; 30 22 496 ; 11 9 372 ; 
18 6 252 ; 19 29 355 ; 27 11 473 ; 0 6 400 ; 25 23 274 ; 
28 11 215 ; 19 2 133 ; 28 8 197 ; 18 20 159 ; 28 29 141 ; 
2 1 384 ; 31 5 43 ; 10 22 306 ; 17 18 370 ; 8 0 174 ; 
20 19 372 ; 22 12 404 ; 31 6 299 ; 19 5 231 ; 16 3 137 ; 
18 11 93 ; 23 6 71 ; 13 21 207 ; 31 31 293 ; 31 9 222 ; 
18 13 363 ; 30 13 55 ; 4 0 489 ; 10 15 356 ; 13 17 356 ; 
24 14 360 ; 31 14 474 ; 20 13 405 ; 19 22 126 ; 28 17 389 ; 
10 20 468 ; 23 11 430 ; 24 24 409 ; 7 5 305 ; 24 5 279 ; 
7 21 324 ; 21 24 473 ; 2 17 268 ; 24 24 220 ; 9 29 199 ; 
29 2 27 ; 8 24 429 ; 1 13 309 ; 6 7 317 ; 11 10 332 ; 
0 27 197 ; 24 4 4 ; 10 12 325 ; 2 8 423 ; 31 23 387 ; 
1 2 293 ; 26 24 206 ; 7 5 331 ; 14 10 8 ; 25 22 446 ; 
20 27 149 ; 24 7 181 ; 4 8 166 ; 12 11 212 ; 3 7 446 ; 
5 12 109 ; 29 14 171 ; 2 25 209 ; 12 5 336 ; 3 23 359 ; 
30 20 56 ; 6 18 456 ; 14 8 32 ; 26 8 200 ; 1 10 236 ; 
13 11 487 ; 27 14 332 ; 21 11 202 ; 26 16 404 ; 18 3 461 ; 
6 15 352 ; 24 15 126 ; 0 7 140 ; 8 3 219 ; 18 11 137 ; 
30 22 430 ; 12 6 95 ; 24 20 338 ; 8 5 491 ; 11 2 266 ; 
26 2 256 ; 9 0 314 ; 16 5 271 ; 19 24 143 ; 30 5 496 ; 
20 19 126 ; 26 14 144 ; 15 4 73 ; 20 16 121 ; 22 22 477 ; 
24 14 148 ; 24 12 246 ; 29 3 375 ; 21 27 67 ; 26 27 78 ; 
13 13 440 ; 27 30 492 ; 0 27 270 ; 16 16 125 ; 6 13 165 ; 
20 21 351 ; 0 23 35 ; 4 2 196 ; 31 29 473 ; 26 11 291 ; 
7 23 433 ; 5 23 183 ; 0 17 486 ; 16 27 34 ; 29 24 170 ; 
18 12 308 ; 9 23 56 ; 11 27 229 ; 8 29 449 ; 19 20 212 ; 
10 5 366 ; 1 4 185 ; 19 31 167 ; 14 25 383 ; 6 4 456 ; 
18 20 361 ; 10 25 100 ; 5 30 269 ; 2 7 41 ; 22 15 185 ; 
28 17 133 ; 0 26 471 ; 31 9 45 ; 25 25 476 ; 29 25 390 ; 
17 22 304 ; 10 6 71 ; 8 0 373 ; 15 5 110 ; 30 18 53 ; 
15 19 47 ; 10 31 323 ; 19 12 64 ; 12 12 459 ; 5 26 44 ; 
28 14 489 ; 2 21 455 ; 2 6 145 ; 7 29 142 ; 26 22 97 ; 
13 5 372 ; 12 17 183 ; 24 29 217 ; 4 8 345 ; 30 13 245 ; 
13 14 227 ; 2 1 355 ; 9 31 377 ; 6 5 416 ; 28 26 11 ; 
1 14 201 ; 18 18 119 ; 15 31 439 ; 24 0 208 ; 5 14 400 ; 
19 1 259 ; 21 16 236 ; 20 1 254 ; 25 21 39 ; 19 17 349 ; 
2 6 441 ; 20 9 363 ; 19 25 85 ; 20 1 377 ; 2 13 197 ; 
3 24 442 ; 19 16 101 ; 20 11 334 ; 10 14 286 ; 27 24 99 ; 
1 29 75 ; 11 4 381 ; 4 30 405 ; 5 7 216 ; 18 8 456 ; 
11 14 74 ; 27 7 149 ; 6 22 184 ; 21 24 261 ; 13 20 443 ; 
10 3 242 ; 15 20 211 ; 13 25 470 ; 20 17 162 ; 28 30 143 ; 
10 12 370 ; 17 30 249 ; 7 2 16 ; 0 27 279 ; 20 30 76 ; 
23 28 461 ; 11 28 375 ; 4 6 330 ; 21 21 115 ; 19 8 107 ; 
0 14 355 ; 30 31 301 ; 0 12 251 ; 28 31 186 ; 26 7 246 ; 
23 28 382 ; 19 27 21 ; 25 10 192 ; 14 5 356 ; 23 21 230 ; 
5 0 417 ; 5 22 197 ; 17 29 71 ; 16 11 20 ; 23 26 212 ; 
19 28 386 ; 14 22 137 ; 24 26 28 ; 30 3 30 ; 19 1 156 ; 
19 18 51 ; 9 4 437 ; 6 19 210 ; 18 3 239 ; 12 15 155 ; 
8 22 182 ; 30 19 337 ; 25 4 439 ; 28 14 376 ; 29 6 262 ; 
15 22 205 ; 19 11 264 ; 7 5 390 ; 10 16 184 ; 26 11 186 ; 
16 29 438 ; 3 2 231 ; 7 21 29 ; 21 1 230 ; 28 11 394 ; 
18 8 399 ; 29 25 367 ; 2 19 494 ; 18 19 353 ; 29 6 168 ; 
26 4 298 ; 28 11 482 ; 
