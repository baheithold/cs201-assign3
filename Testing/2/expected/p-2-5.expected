0: 11
1: 12(11)23 16(11)20
2: 9(12)64 27(16)34
3: 6(9)42
4: 0(6)23 15(6)28 23(6)1
5: 3(23)30 7(23)4 19(15)47 20(15)26
6: 14(20)25 29(3)4
7: 2(29)27 8(14)32 10(14)8 17(29)71 21(29)23 22(14)41
8: 4(8)26 5(8)26 13(17)45 26(8)37 30(2)22
9: 1(4)8 18(5)0 24(4)4 25(5)17 28(26)11
10: 31(24)3
weight: 772
